app/models/grda_warehouse/vispdat/base.rb

```diff
@@ -174,6 +175,13 @@ def notify_vispdat_completed
       end
     end
 
+    def add_to_cohorts
+      return if changes.empty?
```

Comment:
This will need an additional check.  The client should only be added to the cohort if the VI-SPDAT is being submitted (you can edit and save without submitting.)  see `notify_vispdat_completed` for logic.